spec/models/time_entry_spec.rb

```diff
@@ -351,5 +351,68 @@
         it { expect(TimeEntry.visible(time_entry2.user, project).all).to match_array([time_entry2]) }
       end
     end
+
+    context 'calculate dates' do
+      let (:user3) { FactoryGirl.create(:user) }
+      let(:late_time_entry) do
+        FactoryGirl.create(:time_entry, :project => project,
+                           :work_package => work_package,
+                           :spent_on => 2.days.ago,
+                           :hours => hours,
+                           :user => user3,
+                           :rate => hourly_one,
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.